app/views/sessions/_card.html.erb

```diff
@@ -1,35 +1,67 @@
-<div class="w-full p-5 bg-white rounded-lg">
-    <div class="flex flex-row items-center mb-6">
-      <% if speaker.profile.image.attached? %>
-        <%= image_tag url_for(profile.image), class: "text-red w-12 h-12 rounded-full mr-4" %>
-      <% else %>
-        <div class="flex flex-row items-center justify-center w-12 h-12 p-2 mr-4 rounded-full min-w-12 min-h-12 bg-slate-300">
-            <%= image_tag url_for("user_avatar.svg") %>
+<%# TODO: remove when we add support for multiple speakers %>
```

Comment:
I suggest adding a padding/margin between the last card and the bottom navbar
<img width="1057" alt="image" src="https://github.com/user-attachments/assets/cf88e90f-9768-491c-b99e-c1e87c8fe560">